pkg/postgres/pgadmin/admin_utils.go

```diff
@@ -492,3 +496,35 @@ func GetAllDatabases(postgresConfig *postgres.Config) ([]string, error) {
 
 	return clones, nil
 }
+
+// GetServerVersion - returns the version of PostgreSQL in numeric form.
+func GetServerVersion(postgresConfig *postgres.Config) (int64, error) {
+	log.Debugf("GetServerVersion")
+
+	// Connect to different database for admin functions
+	connectPool, err := GetAdminPool(postgresConfig)
+	if err != nil {
+		return 0, err
+	}
+	// Close the admin connection pool
+	defer connectPool.Close()
+
+	// Create a context with a timeout
+	ctx, cancel := context.WithTimeout(context.Background(), PostgresQueryTimeout)
+	defer cancel()
+
+	row := connectPool.QueryRow(ctx, serverVersionNum)
+	var version string
+	if err := row.Scan(&version); err != nil {
+		return 0, err
+	}
+
+	// Parse the string as int64
+	versionNum, err := strconv.ParseInt(version, 10, 64)
+	if err != nil {
+		return 0, err
+	}
+
+	log.Debugf("Server Version = %d.", version)
+	return versionNum, nil
```

Comment:
I'm not sure if it is that important to log only after parsing. If so, then why not:
```suggestion
	log.Debugf("Server Version = %d.", version)
	return strconv.ParseInt(version, 10, 64)
```